docs/other-api/dev.md

```diff
@@ -14,23 +14,70 @@ To get a full list of available commands and flags, run:
 npx @remix-run/dev -h
 ```
 
-## `remix build`
+## `remix vite:build`
 
-Builds your app for production. This command will set `process.env.NODE_ENV` to `production` and minify the output for deployment.
+Builds your app for production with [Remix Vite][remix-vite]. This command will set `process.env.NODE_ENV` to `production` and minify the output for deployment.
+
+```shellscript nonumber
+remix vite:build
+```
+
+| Flag                  | Description                                             | Type                                                | Default     |
```

Comment:
Are these all vite configs?  Should we link off to the source of truth docs for them here as well?